orbit/pkg/table/cis_audit/cis_audit_windows.go

```diff
@@ -0,0 +1,720 @@
+	if constraintList, present := queryContext.Constraints["item"]; present {
+		for _, constraint := range constraintList.Constraints {
+			if constraint.Operator == table.OperatorEquals {
+				inputItem = constraint.Expression // this input as to be kept as-is and returned on the same input column due to a sqlite requirement
+				log.Debug().Msgf("cis_audit input item requested: %s\n", inputItem)
+			}
+		}
+	}
+
+	// Getting the input value if supported
+	if len(inputItem) > 0 {
+		inputValue, err = getValueCisItem(inputItem)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	// returning item and its value
+	return []map[string]string{
+		{
+			"item":  inputItem,
+			"value": inputValue,
+		},
+	}, nil
+}
+
+// getPreProcessedFileContent returns an UTF-16 byte array
+// This is useful when reading data from MS-Windows systems that generate UTF-16BE files
+func getPreProcessedFileContent(path string) ([]byte, error) {
+	// Read the file into a []byte:
+	raw, err := ioutil.ReadFile(path)
+	if err != nil {
+		return nil, err
+	}
+
+	// make an tranformer that converts MS-Win default to UTF8
+	win16be := unicode.UTF16(unicode.BigEndian, unicode.IgnoreBOM)
+
+	// make a transformer that is like win16be, but abides by BOM
+	utf16bom := unicode.BOMOverride(win16be.NewDecoder())
+
+	// make a Reader that uses utf16bom
+	unicodeReader := transform.NewReader(bytes.NewReader(raw), utf16bom)
+
+	// decode and return data
+	decoded, err := ioutil.ReadAll(unicodeReader)
+	if err != nil {
+		return []byte(""), err
+	}
+
+	// replace newlines with unix style
+	fileContent := strings.Replace(string(decoded), "\r\n", "\n", -1)
+
+	return []byte(fileContent), nil
+}
+
+// getSystem32Dir returns the path to the "system32" directory on Windows
+func getSystem32Dir() (string, error) {
+	system32Path, err := windows.GetSystemDirectory()
+	if err != nil {
+		return "", err
+	}
+
+	return system32Path, nil
+}
+
+// getSeceditData returns data from the "secedit.exe" utility
+func getSeceditData() (SeceditData, error) {
+	var data SeceditData
+
+	// Get the path to the system32 directory
+	system32Dir, err := getSystem32Dir()
+	if err != nil {
+		return data, fmt.Errorf("path to system32 could not be determined: %w", err)
+	}
+
+	// Build the fullpath to the "secedit.exe" executable
+	seceditPath := filepath.Join(system32Dir, "secedit.exe")
+
+	// Get temporary directory
+	cacheDir, err := os.UserCacheDir()
+	if err != nil {
+		return data, fmt.Errorf("get UserCacheDir failed: %s", err)
+	}
+
+	// Create temporary directory
+	tempDir, err := ioutil.TempDir(cacheDir, "secedit-")
+	if err != nil {
+		return data, fmt.Errorf("failed to create temporary directory: %w", err)
+	}
+	defer os.RemoveAll(tempDir)
+
+	// Execute "secedit.exe" to export the current security configuration
+	outputInfPath := filepath.Join(tempDir, "output.inf")
+	cmd := exec.Command(seceditPath, "/export", "/cfg", outputInfPath)
+	if err := cmd.Run(); err != nil {
```

Comment:
Are we typically running without context?  ctx is passed down in generate so you could use it here.

Reply:
Question: When running secedit.exe, does it collect all info that we just filter later?

Reply:
> Are we typically running without context?  ctx is passed down in generate so you could use it here.

yes, we have used this to call into os utilities that do not block, examples [here](https://github.com/search?q=repo%3Afleetdm%2Ffleet%20exec.Command&type=code).

Reply:
> Question: When running secedit.exe, does it collect all info that we just filter later?

yes, the idea is to have a single function that deals with secedit output so this is decoupled from the functions consuming this data